src/app/output/output.component.html

```diff
@@ -1,4 +1,4 @@
 <mat-card>
   <mat-card-title>Output</mat-card-title>
-  <textarea class="outputTextBox" rows="20" cols="60" [(ngModel)]="outputString"></textarea>
```

Comment:
I had made comments to remove this in the previous PR. Please make sure they are updated there.